network_diffusion/mln/mlnetwork_torch.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) 2024 by Michał Czuba.
+#
+# This file is a part of Network Diffusion.
+#
+# Network Diffusion is licensed under the MIT License. You may obtain a copy
+# of the License at https://opensource.org/licenses/MIT
+# =============================================================================
+
+"""A converter from `MultilayerNetwork` to the sparse repr. in PyTorch."""
+
+from dataclasses import dataclass
+from typing import Any
+
+import networkx as nx
+import torch
+from bidict import bidict
+
+from network_diffusion.mln.mlnetwork import MultilayerNetwork
+
+
+def _prepare_mln_for_conversion(
+    net: MultilayerNetwork,
+) -> tuple[MultilayerNetwork, bidict, dict[str, set[Any]] | None]:
+    """
+    Prepare `MultilayerNetwork` for conversion to torch representation.
+
+    If network is not multiplex, then multiplicity actoss all layers will be
```

Comment:
(1) actors

Reply:
In fact "across" not "actors", but anyway thanks for spotting this typo